android/src/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -1231,6 +1231,9 @@ public void onChanged(MapObject mapObject)
   @Override
   public void onLocationUpdated(@NonNull Location location)
   {
+    // This could be called before the map object is initialized
+    if (mMapObject == null)
```

Comment:
Why is PP subscribed to location updates at this moment? Wouldn't it be wiser/better/faster/safer to subscribe to location updates when MapObject is set, and unsubscribe when it's cleared?

Reply:
We subscribe to location updates when the fragment is created, at the same time we subscribe to map object updates. But it seems that after an orientation change, the location is updated before the map object. I tried moving the subscription later in the lifecycle but it changed nothing.

A solution would be to subscribe to location change when the map object gets a value for the first time.

Reply:
Found a better way, I simply initialize the mMapObjet value in onCreateView before registering for the location events

Reply:
👍 then these checks are not necessary and can be removed, right?